### ISSUE TITLE
📝 Correction | Re-evaluate links to German, Spanish, French and Polish Translations

### ISSUE THREAD

**Issue by davegson** [privacytools/privacytools.io#2091]
## Description

Remove links to translations of privacytools.io

## Why I am making the suggestion

**Category 1: Outdated** 🇩🇪, 🇫🇷, 🇪🇸

These three languages still have the one-page layout and more importantly, promote vast amounts of VPNs and Mail Services which cannot be good for users. Other categories also obviously got stuck in 2018.

**Category 2: Server Failure** 🇵🇱

https://pl.privacytools.io/ has their SSL configuration messed up somehow, so it currently is inaccessible.

## My connection with the languages

I speak German so can vouch for them being outdated, the others I just evaluated by looking at the images / site design.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

## The Dilemma

I provided a PR but am unsure if you want to follow through with this. I am torn between:

- outdated information may hurt users
- a lot of information still is valid, which might protect users

Let me know what you think!

**Comment by freddy-m** [comment]
> outdated information may hurt users
> a lot of information still is valid, which might protect users 

Very interesting point. If we have any native speakers of these (🇩🇪, 🇫🇷, 🇪🇸) languages, then it would be nice, if you had the time, to work away at slowly updating them. For now, seeing as you've made the PR, I think I'll remove these dead links.

@privacytools/editorial should we create a Matrix room for community translation projects? We've been getting [a](https://forum.privacytools.io/t/would-anyone-be-interested-in-hebrew-translation/4454/9) [few](https://forum.privacytools.io/t/arabic-translation-project-make-a-change/4461) lately?

**Comment by davegson** [comment]
yeah another interesting thing is one can probably also "hotfix" the issues like the russian site did:

they [removed all old VPNs except those you recommend today](https://privacytools.ru/#vpn), however, the [email section still deviates from your current recommendations](https://privacytools.ru/#email]), though not drastically.

the thing with that approach is it will make things a lot more complicated when one wants to catch up with your timeline, since it'll result in two competing branches / timelines.

**Comment by davegson** [comment]
I sadly don't have the capacity at the moment to bring the German site up to speed. I'm more interested in structures and willing to help out there.

- How are translations managed?
- How can people in charge of translation project stay up to date with the current site?

These are probably also valid things to discuss when talking about the new initiatives from the forums

**Comment by freddy-m** [comment]
@davegson I completely understand that you're a busy man, so don't worry that you can't contribute on that front.

> How are translations managed?

I know some translations have their own team members, but I haven't communicated with any of these people for the time that I've been a team member.

> How can people in charge of translation project stay up to date with the current site?

We would have to look into this one. Currently, I'm not sure, however hopefully by discussing it we will come to a solution.

**Comment by dngray** [comment]
> they [removed all old VPNs except those you recommend today](https://privacytools.ru/#vpn),

I find it suspicious NordVPN is listed there in addition to our other VPNs that we recommend on the main site, despite not actually meeting the criteria we set out.

I cannot say I am a fan of that, especially if the translation owner is somehow receiving affiliate money using the PrivacyTools trademark.

I am thinking if we ever do official translations they must be on the privacytools.io domain, ie privacytools.io/ru for example.

I think we were looking for help with https://github.com/privacytools/privacytools.io/issues/1106 (not my area)

**Comment by davegson** [comment]
> I find it suspicious NordVPN is listed there in addition to our other VPNs that we recommend on the main site, despite not actually meeting the criteria we set out.

It's not that they went rogue or anything, it just is outdated. This PTIO VPN list goes way back, and even was a thing until rather recently: a web archive [snapshot from January 2019](https://web.archive.org/web/20190110133012/https://www.privacytools.io/) has the old design with the multitude of VPN providers. Even after the new design the VPNs including NordVPN was up [until August 2019](https://web.archive.org/web/20190805052641/https://www.privacytools.io/classic/). It only [changed in September](https://web.archive.org/web/20190922185516/https://www.privacytools.io/providers/vpn/) when you came out with the new criteria and dumped most of the previously listed providers.

So these translations are just outdated. Proper & up to date translations are a huge undertaking and probably needs more work related to #1106. But the question now is whether to drop the outdated links or not, even more so since you "officially" link to those. But that is your call, not mine.

**Comment by dngray** [comment]
> It's not that they went rogue or anything, it just is outdated. This PTIO VPN list goes way back, and even was a thing until rather recently: a web archive [snapshot from January 2019](https://web.archive.org/web/20190110133012/https://www.privacytools.io/) has the old design with the multitude of VPN providers.

I just found it weird that those 4 were there. We went from a bunch like in the snapshot, to just Mullvad, ProtonVPN and IVPN. This was because these providers had either undergone audits or were in the process of doing so. I found it suspicious that NordVPN was on the russian site, without the others.

![1602940985](https://user-images.githubusercontent.com/48640805/96338261-3e2c1e80-107c-11eb-8e16-7ab344844516.png)

**Comment by davegson** [comment]
interesting! So you mean it can happen that the russian site last got updated just before Nord also got the cut?

**Comment by dngray** [comment]
> So you mean it can happen that the russian site last got updated just before Nord also got the cut?

Yes. if you look at https://github.com/privacytools/privacytools.io/pull/1173 ie the [previous commit](https://deploy-preview-1173--privacytools-io.netlify.app/providers/vpn/#vpn) you can see that it hasn't been updated since August 17th 2019.

**Comment by Booteille** [comment]
Hi.

I translated the french version of the website, and am owning the domain on which this translation is available right now.

During past two years, I tried to check regularly at which state is Weblate support for PTIO (https://github.com/privacytools/privacytools.io/issues/1106) but nothing seems to move.
I don't have a lot of time to give to PTIO, though, and I can't help to move on this subject.

Since PTIO changed a lot, I would probably have to translate again everything from scratch and I would not like to do this work a second time if I know a Weblate support is under construction.

Please, keep me in touch if something move on this subject (either it's getting developed or community decided to not use it in the end).

Thanks for opening this issue.
